voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/VoiceMessageFactory.java

```diff
@@ -113,7 +113,9 @@ private VoiceMessageFactory() {
     }
 
     try {
-      return (VoiceOutput<?>) OUTPUT_MAP.get(type).newInstance();
+      VoiceOutput<?> voiceOutput = (VoiceOutput<?>) OUTPUT_MAP.get(type).newInstance();
+      voiceOutput.setMessage(document);
+      return voiceOutput;
```

Comment:
Oh, you're right.  I had made this change a couple of days ago, and meant to do that, but then when I saw I had an unmerged branch last night I couldn't remember what was left to do, so I opened the PR.